src/cljs/nr/gameboard.cljs

```diff
@@ -1,6 +1,6 @@
 (ns nr.gameboard
-  (:require-macros [cljs.core.async.macros :refer [go]])
-  (:require [cljs.core.async :refer [chan put! <!] :as async]
+(:require-macros [cljs.core.async.macros :refer [go ]])
```

Comment:
Looks like the indentation was missed here, and there's some spacing left after `go`.